modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -132,6 +132,68 @@ users filter ((item, index) -> item.error == null)
 ]
 ----
 
+=== Example
+
+This example reads a JSON array and uses the `filter` function to extract the phone numbers which are active.
+
+==== Input
+[source,JSON, linenums]
+----
+{
+  "Id": "1184001100000000517",
+  "marketCode": "US",
+  "languageCode": "en-US",
+  "profile": {
+  "base": {
+     "username": "TheMule",
+     "activeInd": "R",
+     "phone": [
+       {
+          "activeInd": "Y",
+          "type": "mobile",
+          "primaryInd": "Y",
+          "number": "230678123"
+       },
+       {
+         "activeInd": "N",
+         "type": "mobile",
+         "primaryInd": "N",
+         "number": ""
+       }
```

Comment:
```suggestion
       },
       {
          "activeInd": "Y",
          "type": "mobile",
          "primaryInd": "Y",
          "number": "154896523"
       }
```
With two active phones you could check that the (suggested) script is actually returning both of them.